app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> bool:
+    try:
+        command, file, new_file = command.split(" ")
```

Comment:
no need to use space inside split method, it uses space by default

Reply:
Yes, I know it.  Okey I will change this, but I think it depends on people. I prefer use with quotes